smartsim/settings/mpirunSettings.py

```diff
@@ -242,3 +247,30 @@ def format_env_vars(self):
                 else:
                     formatted += ["-x", name]
         return formatted
+
+
+class MpirunSettings(_OpenMPISettings):
```

Comment:
If this and the following classes are user-facing, can we add a short docstring to explain when they should be used?

Reply:
Frankly I'm embarrassed I forgot to put them in in the first place.

Fixed in [6612e5b](https://github.com/CrayLabs/SmartSim/pull/186/commits/6612e5b2c2ab58017836b81ced8ac7f89ccf4c1d)